packages/zapp/console/src/components/Executions/ExecutionDetails/ExecutionNodeViews.tsx

```diff
@@ -165,6 +156,14 @@ export const ExecutionNodeViews: React.FC<ExecutionNodeViewsProps> = ({ executio
     );
   };
 
+  if (loading) {
+    return (
+      <div className={styles.loading}>
+        <LargeLoadingSpinner />
+      </div>
+    );
```

Comment:
this is an exact copy of `TimelineLoading` defined above, rename `TimelineLoading` to something more generic, ie `LoadingComponent` and call it directly inside `if (loading)`

Reply:
done